core/src/test/java/io/cucumber/core/plugin/DefaultSummaryPrinterTest.java

```diff
@@ -56,17 +57,17 @@ void does_not_print_duplicate_snippets() {
         ));
 
         assertThat(new String(out.toByteArray(), UTF_8), is("" +
```

Comment:
Cheers! I think it would be more readable if `equalToCompressingWhiteSpace` was used here instead of extracting the new line character.

http://hamcrest.org/JavaHamcrest/javadoc/2.1/org/hamcrest/text/IsEqualCompressingWhiteSpace.html